tests/io/test_DFastUtils.py

```diff
@@ -0,0 +1,10 @@
+
+from dfastmi.batch.DFastUtils import get_progloc
+
+class Test_DFastUtils():
+    class Test_get_progloc():
+        def test_get_program_location(self):
+            """
+            Get the location of the program.
+            """
+            assert get_progloc() != ""
```

Comment:
```suggestion

from dfastmi.batch.DFastUtils import get_progloc

class TestGetProgloc():
    def test_get_program_location(self):
        """
        Get the location of the program.
        """
        assert get_progloc().endswith("dfastmi")
```

Reply:
Nice, will update. Thanks for the commit possibility